src/Components/TodoClass.js

```diff
@@ -1,12 +1,22 @@
+import {action, makeObservable, observable} from 'mobx'
 class ToDoClass {
    
     todoArray = [];
+    constructor(){
+        makeObservable(this, {
+            todoArray:observable,
```

Comment:
`todoArray` is not a appropriate name  avoid naming things as class, array or object. the name should automatically suggests its an array.
i.e. in this case you could have used `todos`

Reply:
Ok, Understand